### DISCUSSION TITLE
VSCode cant read path to imask package

### DISCUSSION THREAD

**Discussion by PavelZhuravlyov** [uNmAnNeR/imaskjs#887] [Q&A] [answered]
**Describe the bug**
I am trying to use react-imask code for up myself experience, but VS Code can't read path to inner imask package from react-imask. Please tell me what am I doing wrong? Maybe I forgot make some settings for typescript?

![Снимок экрана 2023-06-26 в 16 59 31](https://github.com/uNmAnNeR/imaskjs/assets/9338039/963f40a7-b312-44dc-848c-cebedc8d09e2)

**Environment:**
 - OS: MacOS Monterey 12.5.1 
 - IMask react-imask 7.0.1

**Comment by uNmAnNeR** [comment, marked as answer]
Hi, @PavelZhuravlyov . If you want to import directly from imask you probably have to install it along with react plugin. for now plugin does not reexport all stuff from imask core.
